shell/platform/fuchsia/flutter/component.cc

```diff
@@ -410,13 +410,6 @@ Application::Application(
   // by decreasing usage counters and flushing code which is still useful.
```

Comment:
Incidentally, is this still an issue?

Reply:
It looks like code collection was removed a while ago, and `--no-collect_code` option is a no-op. Removed `--no-collect_code` as well.